### DISCUSSION TITLE
Why are the anomalous CVD LMS cone fundamentals colour matching functions from "Machado et al. (2009)" not normalized?

### DISCUSSION THREAD

**Discussion by MrColourBlind** [colour-science/colour#843] [General]
Hello,

I am trying to understand the basics of the package, and I am new here :) 

While working with Color Vision Deficiency, I realize that there could be a problem with the CVD package. 
When I shift the L-cone of Stockman & Sharpe cone sensitivities with 20nm with,

`red_blind = colour.msds_cmfs_anomalous_trichromacy_Machado2009(SScmfs, np.array([20, 0, 0]))  
`

the L-cone sensitivity moved to the left side of the spectrum with 20nm which is correct, but the amplitude of it goes beyond 1 which is incorrect. For example, the cone sensitivities of individuals with normal vision are,

![image](https://user-images.githubusercontent.com/88616660/128630994-db237db8-e172-4411-a641-e2ab6b4a9c27.png)

but when I tried to shift it with 20nm it becomes like,

![image](https://user-images.githubusercontent.com/88616660/128631012-c812efb4-23dd-4623-9caa-42b2e13b1b97.png)

So, is there a problem with the package or am I mistaken with the basics of the cone sensitivities?
Could you please help me to understand it?

Sincerely,
MrColourBlind

**Comment by thomasmansencal** [comment]
Hi @MrColourBlind,

This is a great question and it can be puzzling! The rescaling is part of the model, to quote the paper:

```
The replacement model could be simulated simply by replacing the spectral sensitivity function of the L cones with the M cones for the case of protanopia, and the other way around for the case of deuteranopia.
...
The second column of Fig. 8 (Photop. Subst.) illustrates the results obtained with this technique, which, again, are clearly incorrect. This resulted from the fact that, even though the spectral sensitivity functions of all three types of cones had their peak sensitivity independently normalized to 1.0, the areas under these curves are sufficiently different (Fig. 3 left) and need to be taken into account.
...
The rescaling of the curves is performed so that the new curves preserve the areas under the curves of the host cones.
```

Hope it helps!

Cheers,

Thomas

**Reply by MrColourBlind** [reply]
Dear Thomas, 

After re-reading the papers corresponding to this work, your answer rang the bell :) 

Thank you! 👍
